cucumber-rails.gemspec

```diff
@@ -2,7 +2,7 @@
 
 $LOAD_PATH.unshift File.expand_path('lib', __dir__)
 
-Gem::Specification.new do |s|
+Gem::Specification.new do |s| # rubocop:disable Gemspec/RequireMFA
```

Comment:
can we avoid these and have a reason in the standard file.

Also this appears to be new. Should we maybe be doing this?

Reply:
Moved the disabling to `rubocop.yml`. 
I'm not familiar with this setting. I'm not sure what changes are required on the release process and, sadly, I don't have time to investigate. 
I disabled the rule to defer the decision.

My ideas:
1. Defere the introduction of this setting by downgrading rubocop to 1.22.1. RequreMFA [was introduced in 1.23](https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecrequiremfa)
2. Disable RequireMFA rule and make an issue saying that we should deal with it (defere decision but have a reminder)
3. Enable RequireMFA and deal with it on next gem release

I choose number 2 but I can change it. Also open to other suggestions

Reply:
Maybe we should deprecate ruby versions along with rails versions we choose to deprecate?

I wonder if many people runs rails 5.2 with ruby 2.7 ?

Equivalent ruby/rails version, IMO
Rails 4.2 - 2.5 works great, 2.6 needs a hack to run the tests but works great. (1.9.3 is still minimum supported)
Rails 5.x - 2.2.2 becomes minimum, 2.5/2.6 works great obviously.
Rails 6.0 - 2.5 is new minimum, we should be moving to 2.7 soonish
Rails 6.1 - 2.5 remain the minimum, 3.0 is legit candidate
Rails 7 - now requires Ruby 2.7

Reply:
We do this with our running matrix.